inference-engine/src/vpu/graph_transformer/src/stages/static_shape_nms.cpp

```diff
@@ -119,6 +149,21 @@ void FrontEnd::parseStaticShapeNMS(const Model& model, const ie::CNNLayerPtr& la
 
     auto stage = model->addNewStage<StaticShapeNMS>(layer->name, StageType::StaticShapeNMS, layer, usedInputs, DataVector{outIndices, outShape});
     stage->attrs().set<bool>("center_point_box", centerPointBox);
+
+    auto spatDim = inputs[0]->desc().dim(Dim::H);
+
+    const int ALIGN_VALUE = 64;
+    const int buffer_size0 = 2 * sizeof(int16_t) * 4 * spatDim;
+    const int buffer_size1 = 2 * sizeof(int16_t) * spatDim;
```

Comment:
I think we should give more descriptive names for such buffers, at least for now it's not obvious why `buffer_size3` is used in `isCMXEnough` calculation, but not allocated if CMX memory was not enough.

Reply:
add suffixes: ddr, cmx